components/Nav/Nav.js

```diff
@@ -1,7 +1,7 @@
 export const NavLink = React.forwardRef(function NavLink(props, ref) {
   return (
     <a
-      className={`link mr3 black hover-blue${props.active ? " fw7" : " fw5"}`}
+      className={`link mr3 black hover-blue`}
```

Comment:
I pushed a change to make the active link underlined. Maybe it is not best but it will work better with https://github.com/hypered/design-system/issues/19

Reply:
That will do as well, thanks!